lib/handbook/addon/engine.js

```diff
@@ -11,6 +11,8 @@ const Eng = Engine.extend({
     dependencies: {
         services: [
             'analytics',
+            'current-user',
+            'features',
```

Comment:
Hmm, if a component in the handbook uses a feature flag, it'll only be demo'd one way. It might be preferable to demo it both ways, flag on and flag off. But I'm not sure there's a good way to do that...

Cleanest might be using a parameter rather than directly checking the flag, so the parent would do `<NewProjectModal @storageI18nEnabled={{features.storageI18nEnabled}}>` (or whatever) and it can easily be toggled in handbook/tests. Not sure if that's worth it.

Reply:
Ideally this one would be tested both ways, though. What'll be different about the next one?

Reply:
As discussed in person, the main reason not to do this is because the flag won't be disabled in the future and we'll probably remove the check. However, for future components, we'll want to pass in the truthiness of the flag to the component as a feature option, and not try to access the flag from within the component.